Lib/test/test_perf_profiler.py

```diff
@@ -35,17 +35,27 @@ def supports_trampoline_profiling():
 
 
 class TestPerfTrampoline(unittest.TestCase):
-    def setUp(self):
-        super().setUp()
-        self.perf_files = set(pathlib.Path("/tmp/").glob("perf-*.map"))
-
-    def tearDown(self) -> None:
-        super().tearDown()
-        files_to_delete = (
-            set(pathlib.Path("/tmp/").glob("perf-*.map")) - self.perf_files
-        )
-        for file in files_to_delete:
-            file.unlink()
+    #
+    # We've removed the buggy setUp and tearDown methods.
+    # They caused race conditions by scanning /tmp for `perf-*.map` files.
+    # Instead, we now use `addCleanup()` inside each test to delete
+    # the *specific* files we know that test created.
+    #
```

Comment:
This comment should not be kept. It refers to something that doesn't exist anymore.